src/patternfly/components/Dropdown/dropdown.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="pf-c-dropdown{{#if expanded}} pf-is-expanded{{/if}} {{styleModifier}}">
+  <button id="{{id}}" class="pf-c-dropdown__toggle" aria-haspopup="true" aria-expanded="{{#if expanded}}true{{else}}false{{/if}}">
```

Comment:
Should we also be using `aria-controls` and reference the menu's id? I know in the past `aria-controls` had questionable support from screen readers.

Reply:
When I tested `aria-controls` it seems best suited for elements that are not adjacent to the controlling element in the dom. In JAWS, it provides a way to shift focus to that element, and to me seems comparable to an anchor link to another element on the page. I think a better use case for it would be things like toggles or filters on a toolbar that change the view below it.

Something else worth noting, if `aria-controls` is associated with a hidden element, then JAWS will not announce anything related to the `aria-controls` attribute. 

When we reviewed menus with screen reader users, one piece of feedback we received is that focus should automatically be placed on the first menu item. But if it isn't, they had no trouble finding the menu items associated with the menu button. If the dropdown toggle says that it's expanded, they assume the dropdown menu items are the next items that would be announced.